### PR TITLE
Update DMN support in product

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: reference-guide/dmn.adoc [hunk 1]
```diff
@@ -18,7 +18,63 @@ A DMN conformance level 2 implementation includes the requirements in conformanc
 Conformance level 3::
 A DMN conformance level 3 implementation includes the requirements in conformance levels 1 and 2, and supports Friendly Enough Expression Language (FEEL) expressions, the full set of boxed expressions, and fully executable decision models.
 
-{PRODUCT} provides runtime support for DMN 1.1, 1.2, 1.3, 1.4 and 1.5 models at conformance level 3 and design support for DMN 1.5 models at conformance level 3. You can design your DMN models with {CANVAS} or import existing DMN models into your {PRODUCT} projects for deployment and execution. Any DMN models that you import and open in the DMN designer, are converted to DMN 1.5 models.
+{PRODUCT} provides runtime support for DMN 1.1, 1.2, 1.3, 1.4 and 1.5 models at conformance level 3 and design support for DMN 1.2 and 1.5 models at conformance level 3.
```
Comment (gitgabrio) [inline reference-guide/dmn.adoc:21]: Hi @jomarko Thanks for PR. About 
`design support for DMN 1.2 and 1.5 models` I think we should explain (even very briefly, just a sentence) how this different version support is achieved: wdyt ?

Comment (jomarko) [reply reference-guide/dmn.adoc:21]: hmm, my understanding was it is explained in the table below. However, I am fine adding a sentence, but, probably some suggestion for the sentence would help me.

is your point that the sentence should explain 1.2 is achieved using old editor and 1.5 is achieved using the new editor?

Comment (gitgabrio) [reply reference-guide/dmn.adoc:21]: HI @jomarko , yes something like that, even in parentheses.
If a reader start reading the document from top to bottom, without knowing anything else, and get to that sentence, (s)he could be slightly confused

**Review by kbowers-ibm — APPROVED** [review]

**Comment by LightGuard** [comment]
Are we good to merge?

File: reference-guide/dmn.adoc (hunk 1 above)
Comment (jomarko) [reply reference-guide/dmn.adoc]: changes in

**Comment by jomarko** [comment]
@LightGuard yes, from my point of view, if reviewers are fine

**Review by gitgabrio — APPROVED** [review]

**Pull request merged**